Makefile

```diff
@@ -11,8 +11,13 @@ default : install
 build : test
 	CGO_ENABLED=0 go build -trimpath -ldflags "$(LDFLAGS)" .
 
-install : test
+install : build 
```

Comment:
I'm not enough of a gopher so I'll also sub @wlynch in.

My understanding is that `go install` compiles the binary in addition to installing it (with some caching of dependency builds as well). If the install target invokes the build target prior to building it again, are we just duplicating the build?